GNUmakefile

```diff
@@ -10,13 +10,6 @@ GOTOOLS = \
 	github.com/vektra/mockery/cmd/mockery
 
 GOTAGS ?=
-GOMODULES ?= ./... ./api/... ./sdk/...
-GOFILES ?= $(shell go list $(GOMODULES) | grep -v /vendor/)
-ifeq ($(origin GOTEST_PKGS_EXCLUDE), undefined)
```

Comment:
I removed the `GOTEST_PKGS_EXCLUDE` feature completely rather than figure out how to use that to omit some packages from all 3 submodules and then check to see if we can omit one or more of the submodules completely from tests. Seems like you're better off just doing `go test` manually at that point then having a crazy bash one liner nested inside of some make targets.